packages/bpk-component-accordion/src/BpkAccordionItem.scss

```
@@ -74,6 +74,15 @@
     display: inline-block;
   }
 
+  &__leading-icon {
+    margin-right: $bpk-spacing-xs;
+
+    @include bpk-rtl {
+      margin-right: 0;
+      margin-left: $bpk-spacing-xs;
+    }
+  }
+
```

Why not just use the mixin?

```suggestion
  &__leading-icon {
    @include bpk-margin-trailing($bpk-spacing-xs, true);
  }

```

---

Oh I didn't realise there was this mixin over the other one I had used

---

No worries! We added it about a year ago because it was a pattern we were doing a lot

---

Solved!